frontend/src/static/js/components/test/webstatus-overview-filters.test.ts

```diff
@@ -79,5 +79,37 @@ describe('downloadCSV', () => {
     downloadCSVStub.restore();
     toastStub.restore();
   });
-  // TODO. Add a test for the download error.
+
+  it('should display an error toast when the allFeaturesFetcher promise rejects', async () => {
+    const apiClient = new APIClient('');
+    const getAllFeaturesStub = sinon.stub(apiClient, 'getAllFeatures');
+    getAllFeaturesStub.rejects(new Error('Test error'));
+
+    const location = {search: ''};
+    const filterComponent = await fixture<WebstatusOverviewFilters>(
+      html`<webstatus-overview-filters
+        .location=${location}
+        .apiClient=${apiClient}
+      ></webstatus-overview-filters>`
+    );
+    assert.exists(filterComponent);
+    await filterComponent.updateComplete;
+
+    // Click the 'Export to CSV' button
+    const downloadButton =
+      filterComponent.shadowRoot?.querySelector<HTMLButtonElement>(
+        '#export-to-csv-button'
+      );
+    assert.exists(downloadButton);
+
+    const toastStub = sinon.stub(Toast.prototype, 'toast');
+    downloadButton.click();
+    await elementUpdated(filterComponent);
+
+    await waitUntil(
+      () => filterComponent.exportDataStatus,
+      'Export data status failed to change'
+    );
+    expect(toastStub.calledOnce).to.be.true;
+  });
```

Comment:
Nit: To be consistent, could you check the arguments like the other test please?




https://github.com/GoogleChrome/webstatus.dev/pull/636/commits/987ef0b6cf2b56eb66b73c1206b0726d4ddb61ef#diff-de4e062a691241d347f6e4f2d3588b5043e78258485a2042dd16a63521408d33R69-R75

Reply:
Yeah, I was in a rush before heading out for a concert.  Done now.

Reply:
I see the CI is failing. This new test is asserting a different code path (and different toast error) than the first test. You will need to change arguments being checked to match [this different toast](https://github.com/GoogleChrome/webstatus.dev/pull/636/files#diff-a3ad258546614ae5a617d8adacb54dc9cb0f7714c30cdb6d5c445c8db38ca863R260).

Reply:
Fixed. I misunderstood the composition of error message text.

Reply:
done